docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +110,129 @@ your Identity Schema:
-vary: Origin
-vary: Cookie
-strict-transport-security: max-age=15724800; includeSubDomains
-
-<a href="https://playground.projects.oryapis.com/hosted/verification?flow=ba66958a-5f15-4b8f-88f9-52b35020db5c">See Other</a>.
-```
+### Email Templates
 
-The server responds with a HTTP 303 redirect to the Verification UI, appending the `?flow=<flow-id>` query parameter (see the curl
-example) to the URL configured here:
+Ory Identities (Ory Kratos) comes with default email templates for verification flows.
 
-<Tabs
-  defaultValue="oss"
-  values={[
-    {label: 'The Ory Network', value: 'oc'},
-    {label: 'Ory Identities', value: 'oss'}
-  ]}>
-  <TabItem value="oc">
+You can quickly replace the defaults and customize the messages to match the look and feel of your solution. Read the
+[custom email template documentation](../../emails-sms/05_custom-email-templates.mdx) to learn more.
 
-The Ory Network offers a default UI implementation. Visit
-[Bring Your Own UI](https://www.ory.sh/docs/guides/bring-your-user-interface) to learn how to implement a custom UI.
+:::note
 
-  </TabItem>
-  <TabItem value="oss">
+When users attempt to initiate verification for unregistered addresses, the system sends a message to the email address that was
+used in the attempt.
 
-You can configure which recovery URL to use in the Ory Identities config:
+This prevents account enumeration attacks as explained in this
+[blog post by Troy Hunt](https://www.troyhunt.com/website-enumeration-insanity-how-our-personal-data-is-leaked/).
 
-```yaml title="config.yml"
-selfservice:
-  flows:
-    verification:
-      # becomes http://127.0.0.1:4455/verification?flow=df607aa1-d555-4b2a-b3e4-0f5a1d2fe6f3
-      ui_url: http://127.0.0.1:4455/verification
-```
-
-  </TabItem>
-</Tabs>
-
-### Verification for client-side (AJAX) browser clients
+:::
 
-The Verification Flow for client-side browser clients relies on AJAX requests.
+### Require a verified address for login
 
-<AjaxWarning />
+Depending on your use case, you might want to limit what a user without at least one verified address is able to do in your
+application. Ory Identities supports requiring a verified address before the user can log into their account.
 
-<SelfServiceSpaFlow
-  flows={["verification"]}
-  success="Set verified flag and redirect to successful Verification Flow"
-  interactions={['"Verify Email"']}
-/>
+```mdx-code-block
+<Tabs groupId="oc-or-oss">
+  <TabItem value="ocp" label="The Ory Network" default>
+```
 
-To initialize the Verification Flow, call the verification initialization endpoint and set `Accept: application/json`:
+Go to the **Authentication** → **Email Verification** page in the [Ory Console](https://console.ory.sh/guides) and enable **
+Require Verified Address for Login**.
 
+```mdx-code-block
+  </TabItem>
+  <TabItem value="oss" label="Ory Identities">
```

Comment:
This shouldn't be there I think?

Reply:
Hmm, So how would you propose the structure then?